autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -180,9 +187,12 @@ def get_hyperparameter_search_space(
 
         if skip_connection_flag:
 
+            shake_shake_flag = False
             shake_drop_prob_flag = False
             if 'shake-drop' in multi_branch_choice.value_range:
                 shake_drop_prob_flag = True
+            elif 'shake-shake' in multi_branch_choice.value_range:
+                shake_shake_flag = True
```

Comment:
Is the condition for `shake_shake_flag` correct?
I found the hierarchy of those two a bit weird.
It makes sense `shake_drop_prob_flag=False` if `not shake_shake_flag`,
but the other way seems strange.
```suggestion
            shake_drop_prob_flag = 'shake-drop' in multi_branch_choice.value_range
            shake_shake_flag = not shake_drop_prob_flag and 'shake-shake' in multi_branch_choice.value_range
```

Reply:
no actually both shake-shake and shake-drop can be in the value range for the multi branch choice. and I think you are right. There is a bug here it should not be elif but just if 'shake-shake' in multi_branch_choice.value_range.